code/src/Components/ThoughtsList.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import moment from "moment";
+import { HeartIcon } from "./HeartIcon";
+
+export const ThoughtsList = ({ thoughts, onLikes }) => {
+  return (
+    <section>
+      <div className="message-container">
+        <h4>{thoughts.message}</h4>
+        <div className="input-container">
+          <div className="likes-container">
+            <button
+              className={thoughts.hearts > 0 ? 'moreThanZeroClicks' : 'zeroClicks' }
```

Comment:
Cool feature with a grey heart when it´s zero clicks and it turns pink when you like it.